README.md

```diff
@@ -177,8 +212,85 @@ public static void main(String[] args) throws Exception
 }
 ```
 
+## Entity Lifetimes
+
+An **Entity** is the term used to describe types such as **GuildChannel**/**Message**/**User** and other entities
+that Discord provides.
+Instances of these entities are created and deleted by JDA when Discord instructs it. This means
+the lifetime depends on signals provided by the Discord API which are used to create/update/delete entities.
+This is done through Gateway Events known as "dispatches" that are handled by the JDA WebSocket handlers.
+When Discord instructs JDA to delete entities they are simply removed from the JDA cache and lose their references.
+Once that happens nothing in JDA interacts or updates the instances of those entities and they become useless. Discord
+may instruct to delete these entities randomly for cache synchronization with the API.
+
+**It is not recommended to store _any_ of these entities for a longer period of time!**
```

Comment:
We should give guidance on what should be done instead of storing them.
And what storage actually means

e.g: It is fine to store in a local event-driven variable, but not in a long lived class variable or static variable

Reply:
I've see enough people take the advice of "use ids instead of storing entities" and interpret it as "use id lookup literally every time you want to interact with an entity", which of course, isn't the case.

Your example below is a good example of how to properly lookup and store for local var execution.